src/lib/formats/json/parser.ts

```diff
@@ -8,10 +8,11 @@ const ValidationError = require("../../bindings/http/validation/validation_error
 const invalidPayloadTypeError = new ValidationError("invalid payload type, allowed are: string or object");
 const nullOrUndefinedPayload = new ValidationError("null or undefined payload");
 
-const asJSON = (v) => (isString(v) ? JSON.parse(v) : v);
+const asJSON = (v: any) => (isString(v) ? JSON.parse(v) : v);
 
 class JSONParser {
-  constructor(decorator) {
+  decorator: any
+  constructor(decorator: any) {
```

Comment:
The only decorator used at the moment is the `Base64Parser`. Could this be `{}`?

Reply:
I've successfully tested changing this to `Base64Parser` locally.

Reply:
Fixed. Thanks.